submissions/AnnaStoyano/Friends-app/index.js

```diff
@@ -0,0 +1,163 @@
+document.addEventListener('DOMContentLoaded', function () {
+    const API_PATH = 'https://randomuser.me/api/';
+    const USER_COUNT = 20;
+    const URL_FOR_SEARCH = `${API_PATH}?results=${USER_COUNT}`;
+
+    fetch(URL_FOR_SEARCH)
+        .then(resolve => resolve.json())
+        .then(data => {
+            const friendsApp = new FriendsApp(data.results);
+            friendsApp.start();
+        });
+});
+
+class FriendsApp {
+    constructor(users) {
+        this.resultsWrapper = document.querySelector('.result');
+        this.cards = users.map(item => new UserCard(item));
+        this.dipslayCards = this.cards;
+        this.previouseDisplay = this.dipslayCards;
+        this.filterByName();
+        this.filterByAge();
+        this.filterByGender();
+        this.filterByCity();
+        this.filterByEmail();
+    }
+
+    start() {
+        this.render(this.dipslayCards);
+    }
+
+    render() {
+        this.resultsWrapper.innerHTML = '';
+        this.dipslayCards.forEach(({card}) => this.resultsWrapper.insertAdjacentElement('afterbegin', card));
+    }
+
+    filterByName() {
+        this.filterByNameAlphabet();
+        this.filterByNameNonAlphabet();
+    }
+
+    filterByNameAlphabet() {
+        const alphabetBtn = document.querySelector('.name-alphabetically');
+        alphabetBtn.addEventListener('click', () => {
+            this.dipslayCards = this.dipslayCards.sort((a, b) => a.name > b.name ? -1 : 1);
+            this.render();
+        });
+    }
+
+    filterByNameNonAlphabet() {
+        const nonAlphabetBtn = document.querySelector('.name-nonalphabetically');
+        nonAlphabetBtn.addEventListener('click', () => {
+            this.dipslayCards = this.dipslayCards.sort((a, b) => a.name > b.name ? 1 : -1);
+            this.render();
+        });
+    }
+
+    filterByAge() {
+        const ageBtn = document.querySelector('.age_btn');
+        ageBtn.addEventListener('click', () => {
+            if (!ageBtn.classList.contains('LowToHigh')) { // if button is clicked show from low to high
+                this.dipslayCards = this.dipslayCards.sort((a, b) => a.age > b.age ? -1 : 1);
+                ageBtn.classList.add('LowToHigh'); // adding class helps to find out clicking on button
+            } else {
+                this.dipslayCards = this.dipslayCards.sort((a, b) => a.age > b.age ? 1 : -1); // if button isn't clicked show from high to low
+                ageBtn.classList.remove('LowToHigh');
+            }
+            this.render();
+        });
+    }
+
+    filterByGender() {
+        const manRadio = document.querySelector('.radio-input.man');
+        const womanRadio = document.querySelector('.radio-input.woman');
+        const allRadio = document.querySelector('.radio-input.all');
+        const genderWrapper = document.querySelector('.menu_item-gender');
+        genderWrapper.addEventListener('click', () => {
```

Comment:
`input` can have `value`. You can take `value` from checked input and then use it. You will remove your man/woman ifs.